### PR TITLE
Docs and readme.md french translation

### PR DESCRIPTION
**Pull request by CrocodileCroco**

### CODE REVIEW

**Comment by Victrid** [comment]
Looks good to me!

File: readme.fr.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# Splatplost
+
+[中文](readme.zh-CN.md)
```
Comment (Victrid) [inline readme.fr.md]: You can add a link on any `readme.<language>.md` to french version, like:
```suggestion
[中文](readme.zh-CN.md) | [Français](readme.fr.md)
```

File: readme.fr.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+Splatplost est un utilitaire qui vous permet d'imprimer des images dans un dessin Splatoon, basé sur [libnxctrl](https://github.com/Victrid/libnxctrl). Pas comme les micro-controlleurs Teensy, ou les microcontrolleurs AVR, Vous avez seulement besoin d'utiliser le bluetooth de votre machine, ce qui est facile à faire. Avec un algorithme d'impression optimisé, cela peut vous sauver 1/3 du temps d'impression.
+
+## Usage basique
+
+### Installation
+
+Il est recommandé d'utiliser une machine sous Linux. Ou si vous en avez pas, utiliser une [image flashable](docs/image.md). Windows ou MacOS ne sont pas supportés, car libnxctrl est basé sur bluez, le stack bluetooth de Linux.
```
Comment (Victrid) [inline readme.fr.md]: You may need to change the link to the French version of file

```suggestion
Il est recommandé d'utiliser une machine sous Linux. Ou si vous en avez pas, utiliser une [image flashable](docs/image.fr.md). Windows ou MacOS ne sont pas supportés, car libnxctrl est basé sur bluez, le stack bluetooth de Linux.
```

File: docs/image-fr.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# Utiliser une image préconfigurée
```
Comment (Victrid) [inline docs/image-fr.md]: Could you change the filename to make it consistent?
```diff
- docs/image-fr.md
+ docs/image.fr.md
```

**Review by Victrid — CHANGES_REQUESTED** [review]

File: docs/image-fr.md (hunk 3 above)
Comment (CrocodileCroco) [reply docs/image-fr.md]: I changed the filename, however i'm not sure if Github got it right :/

**Review by Victrid — APPROVED** [review]

**Comment by Victrid** [comment]
LGTM, I'll merge it into master branch!

**Pull request merged**